src/components/Person/Person.jsx

```diff
@@ -1 +1,17 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const personPartnerType = person.sex === 'm' ? 'wife' : 'husband';
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">My name is {person.name}</h2>
```

Comment:
Use destructuring for `person`

Reply:
Good evening! In this task instructions we see condition: ''pass the whole person as a prop <Person person={misha} /> (not individual fields)'' Is your comment matches this condition?